xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/pom.xml

```diff
@@ -54,12 +54,6 @@
       <artifactId>xwiki-platform-notifications-notifiers-api</artifactId>
       <version>${project.version}</version>
     </dependency>
```

Comment:
I think you forgot to skip revapi in this module since now it should be executed in the legacy module.